### DISCUSSION TITLE
does openupm supports for checking for updates?

### DISCUSSION THREAD

**Discussion by 1matin** [openupm/openupm#2841] [Ideas]
Hi. does openupm supports for checking for updates? if yes, is it like unity registry packages that let user know if there's an update or like git that should click on update to check for it?
thanks.

**Comment by favoyang** [comment]
Hi @matinmn87,

Thanks for the idea. For now, you can use the [package updates](https://api.openupm.com/feeds/updates/rss) feed for all packages, or GitHub's watch feature to catch up with individual new releases.

If the project gets more support it may be possible to include a user-based notification service for your favorite packages. But not for now.

**Comment by favoyang** [comment]
Converted to a discussion.

**Comment by 1matin** [comment]
is it possible to make a openupm update manager (or sth like that) as a package in unity to manage & check for other openupm packages updates?

**Reply by favoyang** [reply]
It's not possible because of breaking the [Unity Package TOS](https://unity3d.com/legal/terms-of-service/software/package-guidelines?_ga=2.219409177.1207983028.1604503910-134253320.1574534508), [forum thread](https://forum.unity.com/threads/updates-to-our-terms-of-service-and-new-package-guidelines.999940/). We can not write a package to maintain the Unity packman, including the update alert.

**Reply by 1matin** [reply]
so, you can ask for user to install it optionally
